blogs/whats_new_1.7/index.md

```diff
@@ -0,0 +1,153 @@
+
+![Developer perspective - Network Observability](images/developer_perspective-network_traffic.png)
+<div style="text-align: center">Figure 7: Network Observability in Developer perspective</div>
+
+
+## Security features
+
+There are two security-related features in this release.
+
+### FIPS compliance
+
+Network Observability 1.7 is compliant with FIPS 140-3, ensuring that it meets security requirements for cryptographic modules.  This means it can be used when OpenShift is in [FIPS mode](https://docs.openshift.com/container-platform/4.17/installing/overview/installing-fips.html).
+
+### Lock down access to Network Observability
+
+For added security, it is simple to lock down access to Network Observability so only known components can access it.  When you create the FlowCollector resource, enable this feature (Figure 8).  This creates a Kubernetes network policy in the *netobserv* and *netobserv_privileged* namespaces to restrict its access.  Although this is disabled by default, it is recommended to enable this or create your own network policies.
+
+![FlowCollector - Network policy](images/flowcollector-network_policy.png)
+<div style="text-align: center">Figure 8: Network policy</div>
+
+
+## Network Observability CLI enhancements
+
+Network Observability CLI (NetObserv CLI) is a kubectl plug-in that installs a separate Network Observability from the command line and displays a text-based flows table and can also capture packets to a pcap file.  It uses no storage!  When you exit the program, it cleans up and removes itself.
+
+You don't need to install the Network Observability Operator because NetObserv CLI will do it for you.  Just [follow the instructions here](https://docs.openshift.com/container-platform/4.17/observability/network_observability/netobserv_cli/netobserv-cli-install.html) to install NetObserv CLI.  The prerequisite is to have `oc` installed on your computer.
+
+Make sure you have the 1.7 version, which is the GA version.  Enter:
+```
+$ oc netobserv version
+Netobserv CLI version 1.7.0
+```
+
+There were a number of significant changes in NetObserv CLI from the last Network Observability release.  To start, the command line options changed so that when doing packet capture, you now have many of the same options as flow capture, particularly for the eBPF flow filters.  To see these options, enter `oc netobserv help`.  You may need to widen the terminal to view this legibly.
+
+Regardless whether you do flow capture or packet capture, it displays the flows table.  The screen is pretty self-explanatory on what you can do such as filtering data, showing more columns of data, and increasing or decreasing the number of rows.  To exit, press ctrl-c.  It then prompts to save the capture output.  If you respond with 'y', it saves the data with a timestamp filename in the directory **./output/flow/** for flows and **./output/pcap/** for packets.  Figure 9 shows a screenshot of NetObserv CLI after running:
```

Comment:
```suggestion
Regardless of whether you do flow capture or packet capture, you can view the flows table.  The screen is pretty self-explanatory on what you can do such as filtering data, showing more columns of data, and increasing or decreasing the number of rows.  To exit, press <kbd>ctrl</kbd>-<kbd>c</kbd>.  There is a prompt to save the capture output.  If you respond with 'y', it saves the data with a timestamp filename in the directory `./output/flow/` for flows and `./output/pcap/` for packets.  Figure 9 shows a screenshot of Network Observability CLI after running:
```

Reply:
- Instead of using "it", I tried to make the language more focused on what the user would be experiencing/doing.
- I wonder if you can use this html to make the keyboard commands format like keys?
- the filepaths should be in backticks. UI items are typically bolded.

Reply:
The "you can view" changes the meaning.  The intention is to make clear that regardless of whether you do flow capture or packet capture, Network Observability CLI shows you the flows table.  
  
The rest of the changes were made.